docs/README.md

```diff
@@ -0,0 +1,11 @@
+# Pulumi Terraform Bridge Developer Documentation
+
+Welcome to the Pulumi Terraform Bridge developer documentation! This site is designed to provide
+complete documentation for maintainers of and contributors to the Terraform Bridge as well as authors of Bridged Providers.
+
+:::{toctree}
+:maxdepth: 2
+:titlesonly:
+
```

Comment:
I think this needs to explain how to build the docs locally (so we can preview changes).